src/vm/amd64/JitHelpers_InlineGetThread.asm

```diff
@@ -982,6 +986,16 @@ endif
     RetryHelperThinLock:
         jmp     RetryThinLock
 
+	TimeoutZeroThinLock:
+        ; Did not acquire, return FALSE
+        mov     byte ptr [r8], 0
+ifdef MON_DEBUG
+ifdef TRACK_SYNC
+        add     rsp, MON_ENTER_STACK_SIZE_INLINEGETTHREAD
+endif
+endif
+        ret
```

Comment:
This is same as "TimeoutZero:" exit. Would it make sense to jump to "TimeoutZero:" above and not have this part?